eval/README.md

```diff
@@ -0,0 +1,26 @@
+# Evaluation
+
+### Running HEST
+
+1. Install HEST and get the HEST-1k data according to the HEST's manual
+2. Checkout the submodules `git submodule update --init --recursive`
+3. Install hestcore `pip install hestcore`
+4. Run the benchmark: `python run_hest.py --config hest_bench_config.yaml`
+
+### Running EVA
+1. Checkout the submodules `git submodule update --init --recursive`
+2. Copy the custom model definitions [kaiko.py](./eva/src/eva/vision/models/networks/backbones/pathology/kaiko.py) and other helpers to eva: `cp kaiko.py object_tools.py backbones.py renormalized.py ./eva/src/eva/vision/models/networks/backbones/pathology/; mv kaiko.py kaiko.py_;`
+2. Install EVA: `pip install -e eva`
+2. Run benchmarks [run_eva_internal.sh](./run_eva_internal.sh), e.g.:
+```bash
+MODEL_NAME="vitg14_Kaiko_Midnight_concat";
+TASK="camelyon16_small";
```

Comment:
To download the data we need to add the step https://kaiko-ai.github.io/eva/main/datasets/camelyon16/#download-and-preprocessing
Or you can replace it with a task that includes automatic downloading, and add the variable `DOWNLOAD_DATA=true`

Reply:
I added the references to eva's user guides